bedrock/mozorg/templates/mozorg/incognito-browser.html

```diff
@@ -0,0 +1,199 @@
+
+    <img src="{{ static('img/mozorg/technology/incognito-browser-a.png') }}" alt="" class="c-article-image">
+
+    <h2 class="c-section-title">{{ _('What Incognito/Private Mode Does') }}</h2>
+
+    <p>{{ _('Incognito or private mode keeps your browsing history private. That’s it.') }}</p>
+
+    <h2 class="c-section-title">{{ _('What Incognito/Private Mode Doesn’t Do') }}</h2>
+
+    <p>
+    {% trans url='https://www.cnbc.com/2018/07/12/private-web-browsing-not-as-private-as-you-think-research.html' %}
+      A 2018 survey of 460 internet users by the University of Chicago found that
+      <a href="{{ url }}">there are a lot of misconceptions</a> out there about private
+      browsing or incognito mode. It won’t protect you from viruses or malware. It won’t
+      keep your internet service provider (ISP) from seeing where you’ve been online. It
+      won’t stop websites from seeing your physical location. And any bookmarks you save
+      while in private browsing or incognito mode won’t disappear when you switch it off.
+    {% endtrans %}
+    </p>
+
+    <h2 class="c-section-title-large">{{ _('Why go private/incognito?') }}</h2>
+
+    <p>
+    {% trans %}
+      Just because you’re using private browsing mode doesn’t mean you’re up to something
+      nefarious. Perhaps you want to keep your work and personal life separate. You might
+      share a computer or device and you don’t want your siblings snooping. You could be
+      shopping for a gift and you don’t want anything to spoil the surprise. Or maybe you
+      just want to limit the amount of data companies collect about you and you value privacy.
+      Incognito or private browsing mode is mode for any of these scenarios.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Web Tracking') }}</h3>
+
+    <p>
+    {% trans %}
+      A lot of sites keep track of your browsing activity. Most do it to understand if you’re
+      interested in purchasing a product or clicking on an article. They can also do it to help
+      make their sites easier to use. But almost all tracking is done to serve you ads.
+    {% endtrans %}
+    </p>
+
+    <p>
+    {% trans %}
+      Online ads are customized based on your browsing. Been searching for a new pair of sandals?
+      “Shoe Store X” has a great deal for you. The company knows where you’ve been because they
+      dropped a bit of code into your browser called a cookie. The cookie tracks you, and so do
+      Shoe Store X ads.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Cookies') }}</h3>
+
+    <p>
+    {% trans %}
+      Cookies were first used to customize websites, keep track of shopping carts, and maintain
+      online account security, but today most are used to help companies serve targeted ads.
+    {% endtrans %}
+    </p>
+
+    <p>
+    {% trans %}
+      Here’s how it works: You visit a site, an advertiser leaves a  cookie on your browser.
```

Comment:
Nit: extra space between `a  cookie`. Not a big deal but could make l10n troublesome (no idea if there are any plans to localize this page anyway).